apps/prepopulate/data_sample/vocabularies.json

```diff
@@ -3047,5 +3047,21 @@
             "qcode" : {}
         },
         "display_name" : "Countries"
-    }
+    },
+	{
+	    "_id" : "local_domains",
+	    "display_name" : "Local Domains",
+	    "type" : "manageable",
+	    "unique_field" : "domain",
+	    "selection_type" : "do not show",
+	    "items" : [
```

Comment:
shouldn't this be rather empty by default?

Reply:
This is prepopulation script that is used for tests so I think it is ok to have something there.

Reply:
I see, missed that it's data sample